watttime_sdk/api.py

```diff
@@ -284,9 +286,10 @@ def get_forecast_pandas(
             Literal["co2_moer", "co2_aoer", "health_damage"]
         ] = "co2_moer",
         model_date: Optional[Union[str, date]] = None,
+        include_meta: bool = False,
```

Comment:
Could you explain a little bit about why you wanted to add this parameter?

Reply:
reference `get_historical_pandas` method... which has this parameter, so I
am mostly following Sam's lead, but I think it is because you don't really
need the meta info on every line of the pandas df that comes back, so this
gives you the option of not including that.